sale_order_type/models/account_move.py

```diff
@@ -19,7 +19,6 @@ class AccountMove(models.Model):
 
     @api.depends("partner_id", "company_id")
     def _compute_sale_type_id(self):
-        self.sale_type_id = self.env["sale.order.type"]
```

Comment:
Ok, for this.

Apart of that, I think the method does not compute the value for the whole recordset when type not out_invoice or out_refund (should be initialized with False)

Reply:
Yes good point, but the field is computed and stored=True, for all value outside of the filtered is already taken the value in False.